mappings/net/minecraft/enchantment/EnchantmentHelper.mapping

```diff
@@ -3,6 +3,10 @@ CLASS net/minecraft/class_1890 net/minecraft/enchantment/EnchantmentHelper
 		METHOD accept (Lnet/minecraft/class_1887;I)V
 			ARG 1 enchantment
 			ARG 2 level
+	METHOD method_17884 (Ljava/util/Map;Lnet/minecraft/class_2487;Lnet/minecraft/class_1887;)V
+		ARG 2 enchantment
+	METHOD method_22445 getEnchantments (Lnet/minecraft/class_2499;)Ljava/util/Map;
```

Comment:
This is loading enchantments from a list tag to a map of enchantment to level. I actually am not sure if `fromTag` is appropriate here... :thinking: